plugin.video.liveleak/addon.xml

```diff
@@ -0,0 +1,20 @@
+<?xml version="1.0" encoding="UTF-8" standalone="yes"?>
+<addon id="plugin.video.liveleak"
+    name="LiveLeak"
+    version="0.0.7"
+    provider-name="Botster">
+    <requires>
+        <import addon="xbmc.python" version="2.6.0"/>
```

Comment:
Sure but that refers to the internal python API version, which is different for every release. See here: http://kodi.wiki/view/addon.xml